static/js/dialog_widget.js

```diff
@@ -130,7 +130,7 @@ export function launch(conf) {
         } else if (conf.close_on_submit) {
             close_modal();
         }
-        $("#dialog_error").empty();
+        $("#dialog_error").hide();
```

Comment:
I don't understand this change -- can you explain it?  I would expect this to lead to problems where if you retry your operation with the dialog still open, we display an error (but it's invisible because it's hidden).  

In general all the commits in this PR should have commit messages explaining the motivation for the change and any investigation done to determine that it won't create problems.

Reply:
Updated the commit message and PR description.